src/api/sso/src/middleware.js

```diff
@@ -122,18 +119,34 @@ module.exports.createTelescopeUser = function createTelescopeUser() {
       })),
       { returning: 'minimal' }
     );
+  }
+  return result;
+};
+
+module.exports.createTelescopeUser = function createTelescopeUser() {
+  return async (req, res, next) => {
+    const id = getUserId(req);
 
-    if (feedError) {
-      logger.warn(
-        { code: feedError.code, details: feedError.details, id, data: req.body },
-        feedError.message
-      );
-      // Delete the newly created user if their feeds are invalid
-      await supabase.from('telescope_profiles').delete({ returning: 'minimal' }).match({ id });
-      next(createError(400, 'User feed is invalid or already exists'));
+    if (await checkFeedExists(req.body.feeds)) {
+      next(createError(409, 'A Feed URL has already been registered'));
```

Comment:
Can you file a follow-up issue to add a test for this case?  I don't think we have this in the sso e2e tests, and we should.

Reply:
https://github.com/Seneca-CDOT/telescope/issues/2909